packages/admin/cms-admin/src/cronJobs/CronJobs.tsx

```diff
@@ -0,0 +1,113 @@
+import { gql } from "@apollo/client";
+import {
+    LocalErrorScopeApolloContext,
+    MainContent,
+    messages,
+    Stack,
+    Table,
+    TableQuery,
+    Toolbar,
+    ToolbarFillSpace,
+    ToolbarTitleItem,
+    useTableQuery,
+} from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedDate, FormattedMessage, FormattedTime, useIntl } from "react-intl";
+
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { GQLCronJobsQuery } from "../graphql.generated";
+
+const ScopeIndicatorLabelBold = styled(Typography)`
+    && {
+        font-weight: 400;
+        padding: 0 8px 0 4px;
+        text-transform: uppercase;
+    }
+`;
+
+const ScopeIndicatorContent = styled("div")`
+    display: flex;
+    align-items: center;
+`;
+
+const cronJobsQuery = gql`
+    query CronJobs {
+        cronJobs {
+            id
+            name
+            schedule
+            lastScheduledAt
+        }
+    }
+`;
+
+export function CronJobs(): React.ReactElement {
```

Comment:
I'd prefer CronJobsPage to clarify that this is a page that should used inside a Route component